parlai/crowdsourcing/utils/tests.py

```diff
@@ -132,6 +132,64 @@ def _get_channel_info(self):
         else:
             raise ValueError('No channel could be detected!')
 
+    def _register_and_onboard_mock_agents(self, num_agents: int = 1) -> List[str]:
```

Comment:
Yeah, I think this functionality is useful! But because most of this code is duplicated from `._register_mock_agents()`, could you instead enable this behavior with an `assume_onboarding=True` flag perhaps?